astropy/modeling/tests/test_parameters.py

```diff
@@ -754,6 +755,20 @@ def test_raw_value(self):
         param._internal_value = 4
         assert param._raw_value == 4
 
+    if sys.version_info >= (3, 13):
+        dec = pytest.mark.xfail(
+            strict=True,
+            reason=(
+                "this test is known-broken in Python 3.13.0rc1 "
+                "and we don't want it to start passing silently. "
+                "Reference https://github.com/astropy/astropy/pull/16659"
+            ),
+        )
+    else:
+        # fallback to a no-op decorator
+        dec = lambda f: f
+
+    @dec
```

Comment:
Can we just `xfail` with condition like https://docs.pytest.org/en/stable/how-to/skipping.html#condition-parameter instead of invoking `lambda`? Thanks.

Reply:
We already have global xfail_strict setting.

Reply:
I didn't remember this was allowed and couldn't load the documentation initially, thanks for reminding me. Done !